dao/src/main/resources/com/redhat/ipaas/dao/deployment.json

```diff
@@ -498,7 +498,8 @@
       "name": "Timed Pull to Post Example",
       "description": "This is an example of a Timed Pull to Post Integration",
       "configuration": "",
-      "desiredStatus": "Activated",
+      "desiredStatus": "Deactivated",
```

Comment:
@kahboom we've actually setting `desiredStatus` when saving the integration depending on which button you use.  Recent change I made gets this state reflected in the UI though really the state that we should be showing is actually `currentStatus`, not `desiredStatus`.